manim/renderer/renderer.py

```diff
@@ -28,6 +29,7 @@ def __init__(self):
         self.capabilities = [
             (OpenGLVMobject, self.render_vmobject),
             (ImageMobject, self.render_image),
+            (OpenGLMobject, lambda mob: None),
```

Comment:
I'm not a big fan of this in experimental, I would prefer a hard error (XYZ cannot be rendered)

Maybe a marker class is neccessary? Something like
```py
class InvisibleMobject:
    pass

class Camera(OpenGLMobject, InvisibleMobject):
    ...
```

Reply:
While using `lambda mob: None` was admittedly a bad idea, it would be great if we could render the submobjects of any `OpenGLMobject` such as a `Group`.

I was thinking of something like:
```py
	def __init__(self):
        self.capabilities = [
            (OpenGLVMobject, self.render_vmobject),
            (ImageMobject, self.render_image),
        ]

    def render(self, state: SceneState) -> None:
        self.pre_render(state.camera)
        for mob in state.mobjects:
            self.render_mobject(mob)
        self.post_render()

    def render_mobject(self, mob: OpenGLMobject) -> None:
        for MobClass, render_func in self.capabilities:
            if isinstance(mob, MobClass):
                render_func(mob)
                break
        else:
            for submob in mob.submobjects:
                self.render_mobject(submob)
```

Reply:
My concern is that it wouldn't raise any error/message for something we don't support (yet).
Maybe the following changes would work:
- `logger.debug` instead of `logger.warn` (so users don't get bombarded when using `Group`)
- `if not mob.submobjects: logger.warn(f"Cannot render {mob}")`

Reply:
I made an implementation, and I would like you to take a look, please.
I'm still using `logger.warning`, but I implemented an `._unsupported_mob_warnings` attribute to store warning messages and avoid spamming the console on every frame.
I marked `Group` as `InvisibleMobject`, so no warnings would be raised for it unless any of its submobjects is not renderable and is not an `InvisibleMobject` itself.
Any other `OpenGLMobject` will have a warning raised. IDK about raising a hard error, though. If we want to go down that route, maybe we should enforce that `Mobject`s can't have submobjects unless they're explicitly `Group`s, but that might be controversial and should be done in another PR.